src/app/user-page/user-page.component.ts

```diff
@@ -76,5 +82,14 @@ export class UserPageComponent extends Base implements OnInit {
 
   ngOnInit(): void {
     this.getUserInfo(this.username);
+    this.tokenQuery.tokens$.pipe(takeUntil(this.ngUnsubscribe)).subscribe((tokens: TokenUser[]) => {
+      for (const account of this.accountsInfo) {
+        const found = tokens.find((token) => token.tokenSource === account.source);
+        if (found && !['GitHub', 'Google', 'ORCID'].includes(account.name)) {
```

Comment:
Instead of using strings where there is the possibility of a typo mismatch, you could use/rename the unused `show` property, after updating its declarations accordingly. It also makes is easy to add new accounts that we might want to show without needing to update this code. Something like `const found = tokens.find((token) => token.tokenSource === account.source && account.showForAdmins)`.

I know I keep harping on using types instead of strings, and while you don't have a typo mismatch here, it's better to have the compiler do the check than manually verifying it. Plus, if they ever change the spelling/casing, it's easier to update in one place rather than figure out this code is related to AccountInfo.